<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Farn</class>
 <widget class="QWidget" name="Farn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Farn</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QSlider" name="lengthSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="sliderPosition">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="angleSlider">
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="sliderPosition">
      <number>-90</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="angleLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lengthLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGraphicsView" name="treeView"/>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QPushButton" name="drawButton">
     <property name="text">
      <string>Go!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
